includes/analytics/class-analytics.php

```diff
@@ -73,14 +82,13 @@ public static function get_prophecy_token_header( $args = [] ) {
 		}
 
 		$site_url     = str_replace( array( 'http://', 'https://', 'www.' ), array( '', '', '' ), $site_url );
-		$current_user = wp_get_current_user();
 		$site_name    = get_bloginfo( 'name' );
+		$api_details  = get_option( 'kt_api_manager_kadence_gutenberg_pro_data', [] );
```

Comment:
Safer to just type cast this, in case stored data is bad somehow.

```suggestion
		$api_details  = (array) get_option( 'kt_api_manager_kadence_gutenberg_pro_data', [] );
```